src/index.ts

```diff
@@ -3,6 +3,7 @@ export * from "./frontends/leaflet";
 export * from "./symbolizer";
 export * from "./task";
 export * from "./default_style/style";
+export * from "./default_style/themes";
```

Comment:
These are copied from https://github.com/protomaps/basemaps/tree/main/styles and I would prefer you import that library instead of exposing them here - I'll likely change `protomaps-leaflet` to depend on `protomaps-themes-base` to avoid the copying of the `Theme` interface and 5 default implementations.